recipes/coverage-fixpaths/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "coverage-fixpaths" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18880405439c1a6329e5448b104c2b03de248fa4f54a7ec2f123fd226a2a573b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - coverage-fixpaths = coveragefixpaths.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
```

Comment:
Sorry, I forgot one thing. We should also add a `commands` test to check the cli:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Ah again good catch. Sniped!